tools/test-tools/package.json

```diff
@@ -32,19 +33,18 @@
 	},
 	"devDependencies": {
 		"@fluidframework/build-common": "^2.0.3",
-		"@fluidframework/build-tools": "^0.44.0",
-		"@fluidframework/eslint-config-fluid": "^5.1.0",
-		"@types/mocha": "^10.0.0",
+		"@fluidframework/build-tools": "^0.51.0",
+		"@fluidframework/eslint-config-fluid": "^5.4.0",
+		"@types/mocha": "^9.1.1",
```

Comment:
Update dependencies to match the versions used in the rest of the client release group. Note that it meant a few deps went to an earlier version, but should be ok.